carbon_policy_replicator/README.md

```diff
@@ -1,77 +1,76 @@
-  * To check your configuration, click **Get Policy Info** button. The policy and rule configuration for the organization should load.
-* Add the **Import Orgs** credentials
-  * Click the Settings icon in the top-right corner of the **Import Orgs** window. Click `Add Org` and insert the required `Org Key`, `URL`, `API ID` and `API Secret Key`. Repeat this step for as many organizations as you need, and click `Save`.
-  * To check your configuration, click **Get Policy Info** button in the **Import Orgs** window. The policy and rule configuration for the organizations that you set up should load.
-
-## Using the Tool
-After you are done [Setting up the Tool](#setting-up-the-tool), you can move to reviewing or replication policy information.
-* To view the current policy setup, click the `Get Policy Info` button in either **Export Org** or **Import Orgs** window.
-* To replicate a policy, you need to select atleast one policy from the **Export Org**, then click the `Import Data` button. After you confirm your input in the pop-up prompt, the policy and its selected rules(selected rules) will be replicated in all the **Import Orgs** that you setup.
-
-## Resources:
-Carbon Black Cloud Python SDK
-* [PyPi](https://pypi.org/project/carbon-black-cloud-sdk/)
-* [Github](https://github.com/carbonblack/carbon-black-cloud-sdk-python/tree/master)
-* [Read the Docs](https://carbon-black-cloud-python-sdk.readthedocs.io/)
-
-Developer Network
-* [Developer Network](https://developer.carbonblack.com)
-* [Policy Service API](https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/policy-service/)
-
-Postman
-* [CBC Postman Collection](https://documenter.getpostman.com/view/19038029/2s8YK4to5o#intro)
-
-Have questions or feedback?
-[Contact Us](https://developer.carbonblack.com/contact).
+ # Carbon Policy Replicator
+ Replicate policies and their rules to an unlimited number organizations across different Environments.
+
+ The `Carbon Policy Replicator` is a user-friendly Python-based GUI tool that easily transfers policies and rules from one organization to
+ numerous others. This tool can replicate one or more policies with just a few clicks or save your data so that you can return to it later.
+
+ Under the hood, this tool uses the `Carbon Black Cloud Python SDK` for things like Authentication and Credentials Handling, Retrieving Data from Carbon Black Cloud and Handling Errors and Exceptions. This is one example of how the SDK can be used to build powerful tools and integrations to streamline your workflows.
+
+ **Latest Version:** 1.0.0
+
+ **Release Date:** January 17, 2023
+
+ **Supported OS:** MacOS, Linux
+
+ ## App Features
+
+ * Replicate policies and rules from one organization to any number of other organizations across multiple environments
+ * View multi-org policy and rule information from a single source
+ * Compare Policy information between organizations (coming in next release)
```

Comment:
Let's put a heading after View multi-org policy ...

## Future extensions
* the two already listed plus
* Copy Core Prevention and Host Based Firewall rules